go.mod

```diff
@@ -1036,6 +1073,19 @@ retract v0.37.0 // Contains dependencies on v0.36.0 components, which should hav
 // see https://github.com/distribution/distribution/issues/3590
 exclude github.com/docker/distribution v2.8.0+incompatible
 
-// some dependencies attempt to bring something like v1.8.2-0.20220303173753-edfe657b5405, which is older than v0.38.0
-// at the time of this inclusion, v0.38.0 was the latest version available (also tagged as v2.38.0)
-replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.38.0
+// some dependencies attempt to bring something like v1.8.2-0.20220303173753-edfe657b5405, which is older than v0.39.1
+// at the time of this inclusion, v0.39.1 was the latest version available (also tagged as v2.39.1)
+replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.39.1
```

Comment:
created PR to loki to get rid of this replace https://github.com/grafana/loki/pull/7697
As soon as it is merged I will remove replace from the collector. But I think it is ok to do it in the separate PR, since this replacement is already present in the main branch

Reply:
Sure, this shouldn't block this PR,  let's just create an issue to track this and we can mark it as solved

Reply:
https://github.com/grafana/loki/pull/7697 was merged, so I removed prometheus replace directive from go.mod in this PR instead of creating the new one